src/actions/createArticles.js

```diff
@@ -43,19 +42,17 @@ export const createArticleFailure = message => ({
 
 const createArticle = (props, requestData) => async (dispatch) => {
   dispatch(createArticleLoading());
-  try {
-    const { data } = await axios.post(
-      `${apiUrl}/articles`,
-      requestData.formData,
-      {
-        headers: { Authorization: `Bearer ${requestData.token}` }
-      }
-    );
-    props.history.push(`/article/${data.data.slug}`);
-    dispatch(createArticleSuccess(data.message));
-  } catch (error) {
+  return request({
+    route: 'articles',
+    method: 'post',
+    payload: requestData.formData,
+    token: requestData.token
```

Comment:
#### Nitpick
- Don't pass the token as a parameter in here.

Reply:
@aanchirinah Thank you for the gentle reminder.